sysinfo.ts

```diff
@@ -0,0 +1,99 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+import os = require("os");
+import osenv = require("osenv");
```

Comment:
osenv does not have definition in common - we should move the definition file here.